### DISCUSSION TITLE
How to choose 'content' instead of 'summary' as 'description'?

### DISCUSSION THREAD

**Discussion by onnyyonn** [osmoscraft/osmosfeed#54] [Q&A] [answered]
My rss feed has both 'summary' and 'content' fields. Currently 'description' shows the 'summary'. Is there a way to choose?

**Comment by chuanqisun** [comment, marked as answer]
Not at this point. The `description` field was actually meant for showing the summary. ([see code](https://github.com/osmoscraft/osmosfeed/blob/master/packages/cli/src/lib/enrich.ts#L19)) so it has the correct behavior. I plan to have a separate field for the full `content` field. Definitely a goal for the v2 major release.
